### PR TITLE
add get_job_params endpoint

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

**Comment by Tianhao-Gu** [comment]
@bio-boris mind taking a look?

File: lib/execution_engine2/utils/SDKMethodRunner.py [hunk 1]
```diff
@@ -261,3 +264,45 @@ def run_job(self, params, ctx):
...
+
+            job_params["method"] = job_input.method
+            job_params["params"] = job_input.params
+            job_params["service_ver"] = job_input.service_ver
+            job_params["app_id"] = job_input.app_id
+            job_params["wsid"] = job_input.wsid
+            job_params["parent_job_id"] = job_input.parent_job_id
+            job_params["source_ws_objects"] = job_input.source_ws_objects
+
+        return job_params, job_config
```
Comment (bio-boris) [inline lib/execution_engine2/utils/SDKMethodRunner.py]: job_config unused?

File: lib/execution_engine2/utils/SDKMethodRunner.py [hunk 2]
```diff
@@ -261,3 +264,45 @@ def run_job(self, params, ctx):
...
+
+        with self.get_mongo_util().me_collection(self.config["mongo-jobs-collection"]):
+
+            try:
+                job = Job.objects(id=job_id)[0]
+            except Exception:
+                raise ValueError("Unable to find job:\nError:\n{}".format(traceback.format_exc()))
+
+            job.status = status
+            job.updated = datetime.utcnow()
```
Comment (bio-boris) [inline lib/execution_engine2/utils/SDKMethodRunner.py]: Maybe we should edit the model so we don't have to do job.updated manually each time

https://stackoverflow.com/questions/12230022/update-datetime-field-in-mongoengine-with-auto-now-add

File: lib/execution_engine2/utils/SDKMethodRunner.py (hunk 1 above)
Comment (Tianhao-Gu) [reply lib/execution_engine2/utils/SDKMethodRunner.py]: it's not used. I removed it from spec.

File: lib/execution_engine2/utils/SDKMethodRunner.py (hunk 2 above)
Comment (Tianhao-Gu) [reply lib/execution_engine2/utils/SDKMethodRunner.py]: 👍 sounds like a good idea. Implemented auto update upon saving.

**Pull request merged**